### ISSUE TITLE
Two door bells share the same setup code and setup for chime

### ISSUE THREAD

**Issue by juangamnik** [Samfox2/homebridge-videodoorbell#87]
I installed your plugin and tried to install two cams with one platform (and one child bridge) as well as two platforms and child bridges. Both works, but the setup code for the cameras is always the same and the settings for chime (so which HomePods should play a sound when the door bell is pressed) is changed always for both door bells/cameras. 

What I try to achieve is, that I have one camera/door bell for the day and one for the night. During the day the day-camera is triggered and plays the chime. During night the night-camera is triggered, which does not play a chime. But since the config changes for both cameras when I change it on one or the other, that does not work.

Can you help me, please?

**Comment by Samfox2** [comment]
This is a use case I never thought about but it really makes sense. Have you tried to add the cameras as two separate video doorbells?

**Comment by juangamnik** [comment]
Yes. I even tried to realize it as separate platforms/child bridges. Same behavior. I am not completely sure whether this is an issue of the plug-in or a constraint of HomeKit. But I cannot check it with a real video doorbell as I have no. 

Currently I circumvent the issue by adding a virtual motion sensor for the night and trigger depending on the time either the video doorbell or the motion sensor… so in the night the message of the push notification is a littlebit misleading as it says that it detected „motion of ‚doorbell at night’“, but hey nothing is perfect ;).
